src/plugins/kibana_overview/public/components/overview/overview.test.tsx

```diff
@@ -148,27 +124,94 @@ const mockFeatures = [
   },
 ];
 
+const flushPromises = async () => {
+  await new Promise<void>(async (resolve, reject) => {
+    try {
+      setTimeout(() => resolve(), 10);
+    } catch (error) {
+      reject(error);
+    }
+  });
+};
+
+const updateComponent = async (component: ReactWrapper) => {
+  await act(async () => {
+    await flushPromises();
+    component.update();
```

Comment:
Need to use that for the `indexPatternService.hasUserDataView` promise to resolve.